CONTRIBUTING.md

```diff
@@ -135,9 +135,9 @@ can create and configure the Python virtual environment with these
 commands:
 
 ```console
-cd test
-pyenv virtualenv <python_version_to_use> test
-pyenv local test
+cd skeleton-golang-package
+pyenv virtualenv <python_version_to_use> skeleton-golang-package
+pyenv local skeleton-golang-package
```

Comment:
I don't know much about golang- does it still make sense to use pyenv here?  I'm assuming we can still use it for pre-commit?  If so, we should include some explanation about this.

Is there a golang analog to pyenv that we should be using instead or in addition to pyenv?

Reply:
Missed this. Yeah initially when I generated the skeleton I wanted to remove all the Python related files but realized some are dependent on pre-commit. But you're right this does not belong here.

Reply:
I've re-read and it seems these instructions are crucial for pre-commit

Reply:
Yeah, I think this is required for pre-commit and cannot be removed.

Reply:
As long as this is needed for pre-commit, it makes sense to leave it in here.

For the other part of my question ("is there a pyenv equivalent for Golang?"), it appears that there have been some tools in the past ([goenv](https://pypi.org/project/govenv/), [conda](https://ostechnix.com/create-golang-virtual-environments-using-conda-in-linux/)), but the general consensus (from my brief research here) is that virtual environments are not really necessary with Go:
* https://eli.thegreenplace.net/2020/you-dont-need-virtualenv-in-go/
* https://stackoverflow.com/questions/61813503/is-there-a-go-equivalent-to-pythons-virtualenv

So unless anyone has anything else to add here, I can resolve this conversation with no changes necessary.

Reply:
Doesn't [the Nix package manager](https://nixos.org/) (already being used in this PR) scratch part of that itch?  I thought that was why @jasonodoom is using it here.

Reply:
@dav3r correct. 

@jsf9k that's not really why I'm using flake.nix though it provides that options as a result. It's just a nice way to get started quickly if you don't want to install or set-up Go. And Nix precedes the OS so it's generally understood that Nix is not NixOS. Are you okay with this?